evap/evaluation/models.py

```diff
@@ -1028,19 +1028,23 @@ def __render_string(cls, text, dictionary):
         return Template(text).render(Context(dictionary, autoescape=False))
 
     @classmethod
-    def send_to_users_in_courses(self, template, courses, recipient_groups):
+    def send_to_users_in_courses(self, template, courses, recipient_groups, cc):
```

Comment:
defining whether an email may be sent to other users in cc. if it's False, nobody will receive the email in cc. if it's True, delegates and cc users will receive a copy of the message if such users exist.